.eslintrc.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
           // Common small numbers
           -2, -1, 0, 1, 2, 3,
           // HTTP statuses
-          200, 201, 202, 204, 300, 301, 400, 401, 403, 404, 410, 422, 429, 500,
+          200, 201, 202, 204, 300, 301, 302, 303, 304, 305, 306, 307, 308, 400, 401, 403, 404, 410, 422, 429, 500,
```

Comment:
[sand] Maybe we can enumerate those via something like https://github.com/prettymuchbryce/http-status-codes?

Reply:
Good idea :+1: 
Fixed.